src/js/components/tasks/cards/card.tsx

```diff
@@ -0,0 +1,211 @@
+  const isRefreshing = Boolean(org?.currently_refreshing_changes);
+
+  // If (somehow) there's an org owned by someone else, do not show org.
+  if (ownedByWrongUser) {
+    logError(
+      'A scratch org exists for this task, but is not owned by the assigned user.',
+      {
+        org,
+        assignedUser,
+      },
+    );
+    org = null;
+  }
+
+  const [assignUserModalOpen, setAssignUserModalOpen] = useState(false);
+  const openAssignUserModal = () => {
+    setAssignUserModalOpen(true);
+  };
+  const closeAssignUserModal = () => {
+    setAssignUserModalOpen(false);
+  };
+
+  const doAssignUser = useCallback(
+    (assignee: GitHubUser | null) => {
+      closeAssignUserModal();
+      handleAssignUser({ type, assignee });
+    },
+    [handleAssignUser, type],
+  );
+  const doCreateOrg = useCallback(() => {
+    handleCreate(type);
+  }, [handleCreate, type]);
+  const doDeleteOrg = useCallback(() => {
+    const orgToDelete = org || ownedByWrongUser;
+    /* istanbul ignore else */
+    if (orgToDelete) {
+      handleDelete(orgToDelete);
+    }
+  }, [handleDelete, org, ownedByWrongUser]);
+  const doCheckForOrgChanges = useCallback(() => {
+    /* istanbul ignore else */
+    if (org) {
+      handleCheckForOrgChanges(org);
+    }
+  }, [handleCheckForOrgChanges, org]);
+
+  const handleEmptyMessageClick = useCallback(() => {
+    history.push(projectUrl);
+  }, [projectUrl]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const heading =
+    type === ORG_TYPES.QA ? i18n.t('Reviewer') : i18n.t('Developer');
+  const orgHeading =
+    type === ORG_TYPES.QA ? i18n.t('Review Org') : i18n.t('Dev Org');
+  const userModalHeading =
+    type === ORG_TYPES.QA
+      ? i18n.t('Assign Reviewer')
+      : i18n.t('Assign Developer');
+
+  return (
+    <div
+      className="slds-size_1-of-1
+        slds-large-size_1-of-2
+        slds-p-around_x-small"
+    >
+      <Card
+        className={classNames({ 'has-nested-card': assignedUser })}
+        bodyClassName="slds-card__body_inner"
+        heading={heading}
+        headerActions={
+          <UserActions
+            type={type}
+            assignedUser={assignedUser}
+            openAssignUserModal={openAssignUserModal}
+            setUser={doAssignUser}
+          />
+        }
+        footer={
+          <Footer
+            org={org}
+            ownedByCurrentUser={ownedByCurrentUser}
+            isCreating={isCreating}
+            isDeleting={isDeleting}
+            isRefreshing={isRefreshing}
+          />
+        }
+      >
+        {assignedUser && (
+          <div className="slds-m-bottom_small">
+            <UserCard user={assignedUser} className="nested-card" />
+          </div>
+        )}
+        {(assignedUser || ownedByWrongUser) && (
+          <>
+            <hr className="slds-m-vertical_none" />
+            <Card
+              className="nested-card"
+              bodyClassName="slds-card__body_inner"
```

Comment:
I don't think we want this here as it adds an indent to the info. I'm not sure if it only adds padding, or if it does additional things that are necessary. If it is padding only, we can remove it.